Lib/logging/handlers.py

```diff
@@ -28,6 +28,7 @@
 import queue
 import threading
 import copy
+import warnings
```

Comment:
It is better to defer the `warnings` module import until it is needed (which never happen in normal case).